cves/kernel/CVE-2013-2140.yml

```diff
@@ -452,7 +491,11 @@ mistakes:
 
     Write a thoughtful entry here that people in the software engineering
     industry would find interesting.
-  answer:
+  answer: |
+    It seems that this vulnerability was the result of overlooking error
+    paths. There was no evidence prior to the fix that the code was attempting
+    to prevent sectors being discarded on a read only device, nor was it
+    checking that the sectors being discarded actually existed.
```

Comment:
Seems like the developers of the code that caused the vulnerability made a lot of big assumptions that caused security issues.